src/dom.js

```diff
@@ -938,6 +938,9 @@ export function domOrderComparator(element1, element2) {
  * @return {boolean} Whether or not the element now has the attribute.
  */
 export function toggleAttribute(element, name, forced) {
+  if (IS_ESM) {
+    return element.toggleAttribute(name, forced);
```

Comment:
WDYT about moving this to a separate polyfill so the codebase can use toggleAttribute throughout?

Reply:
(looks like a few files in amp-story code already use it too)

Reply:
Counter-point to myself: this minifies better as a single function call versus actually including `toggleAttribute` everywhere, which can't be minified

Reply:
I was hoping to keep this PR smaller in scope, but yes to polyfilling only where necessary and calling the native method name in a future change.

Reply:
If you file the issue I'll take care of it